.gitignore

```diff
@@ -7,6 +7,7 @@ phpunit.xml
 uploads
 upgrade
 cache
+package-lock.json
```

Comment:
Hey @bmarshall511 👋 

(Thank you for diving in head first and directly starting to contribute back to our tooling :))

This here is an annoying piece thing about the setup we currently have with the scaffold... We want the packages to be on the latest version when the project is initiated. Therefore we don't want to include the lock file here in the `wp-scaffold` template repository. 

But we do want every project to use lock files.

Because of that we cannot add the `package-lock.json` file to the `.gitignore` here. 

I would love to find a better way to make it easier for maintainers to automatically not include the lock file but still ensuring that actual projects then use the lock file. So if you have any ideas we are always open to new ways of doing this :) 

For now please remove this addition here.

Reply:
Completely understand, I've got that updated so it won't be ignored.